package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "osrm",
-  "version": "5.11.0",
+  "version": "5.11.0-latest.1",
   "private": false,
```

Comment:
But we have that problem already - we often merge breaking data changes to `master` and we don't bump the version number.

IMO the biggest concern with "master datasets are all compatible" would be bizzaro tickets being opened by folks not realizing that they're trying to use old data, creating weird error situations and undefined behaviour that's hard to debug.  You're right, I don't think we want to create that kind of trouble for ourselves.

The old approach of checksumming the critical IO files was semi-workable, as long as we can capture all code that affects the file format in the checksum value.  I haven't looked at our re-factoring for a while, are we closer, or further away from being able to automatically detect file compatibility using that approach?